tests/unit/validation/test_yaml_allowlist.py

```
@@ -0,0 +1,319 @@
+
+            for node in ast.walk(tree):
+                if isinstance(node, ast.FunctionDef):
+                    all_functions[node.name] = (full_path, node)
+
+        # Check each allowed function
+        for func_name in allowed_functions:
+            if func_name not in all_functions:
+                continue  # Skip non-existent functions (caught by other test)
+
+            file_path, func_node = all_functions[func_name]
+            uses_yaml, yaml_calls = self._function_uses_yaml_safe_load(func_node)
```

Ruff (0.14.5)

Unpacked variable `yaml_calls` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)